composer.json

```
@@ -18,7 +18,7 @@
     "require": {
         "php": "^8.2",
         "brick/math": "^0.9.3",
-        "nuwave/lighthouse": "dev-master"
+        "nuwave/lighthouse": "^5.71.0"
```

It should be safe to use this:
```suggestion
        "nuwave/lighthouse": "^6.0.0"
```

We'd then need to release a new version, and update to `6.0.0-beta.1` in platform.